src/etc/gdb_load_rust_pretty_printers.py

```diff
@@ -9,4 +9,4 @@
 # except according to those terms.
 
 import gdb_rust_pretty_printing
-gdb_rust_pretty_printing.register_printers(gdb.current_objfile())
+gdb_rust_pretty_printing.register_printers(gdb)
```

Comment:
This is problematic since we deliberately want to register the pretty printers only for Rust objects. With this change GDB would try to also use the pretty printers when stepping into code compiled from other languages. Is this really needed?